digital_assets/README.md

```diff
@@ -0,0 +1,26 @@
+# ASWF Digital Assets License 
+
+This folder contains a license for use with for assets intended to be used solely for any of the following purposes:
+
+- education
+- training
+- research
+- software and hardware development
+- performance benchmarking (including publication of benchmark results and permitting reproducibility of the benchmark results by third parties)
+- software and hardware product demonstrations.
+
+Version 1.0 of this license was approved for use by the Academy Software Foundation Governing Board on 2021-07-29. This license is available to used by both ASWF and non-ASWF projects.
+
+## License Attribution
+
+Assets should include language within it's metadata indicating the use of this license and pointing to the full license text. Example below:
+
+```
+ASWF Digital Assets License v1.0 (available at https://aswf.io/licenses/aswf_digital_assets_license_v1.0.txt)
+```
+
+You can also use the SPDX Short Form identifer as shown below:
```

Comment:
I'd add a bit more detail here -- to say something like:

You can also use a license identifier in the custom SPDX "LicenseRef-" format, such as the following. Please note that this license is _not_ currently on the [SPDX License List](https://spdx.org/licenses), so this identifier uses the "LicenseRef-" prefix and will not be independently recognized by scanning tools.